terraform/azure.tf

```diff
@@ -635,6 +635,68 @@ resource "azurerm_sql_database" "sql-database" {
   tags {}
 }
 
+
+resource "random_string" "mysql_server" {
+  length  = 10
+  special = false
+  upper   = false
+}
+
+resource "azurerm_mysql_server" "mysql" {
+  name                = "${random_string.mysql_server.result}-mysqlsvr"
+  location            = "westeurope"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
+
+  sku {
+    name     = "B_Gen4_2"
+    capacity = "2"
+    tier     = "Basic"
+    family   = "Gen4"
+  }
+
+  storage_profile {
+    storage_mb            = "5120"
+    backup_retention_days = "7"
+    geo_redundant_backup  = "Disabled"
+  }
+
+  administrator_login          = "${terraform.workspace}"
```

Comment:
Please change this to simply `inspec-azure` - logins have max char of 16 and some workspaces will fail. Can you also please change it on line 625 in the `azurerm_sql_server` resource?